src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx

```diff
@@ -36,6 +37,20 @@ const WithIconTemplate: StoryFn<BasicCardProps> = (args) => (
     </div>
 );
 
+const WithIconLeftTemplate: StoryFn<BasicCardProps> = (args) => (
```

Comment:
I think that would be nice to do one tab `with Icon` and show there default position and left position

Reply:
Ok